libkpod/config.go

```diff
@@ -0,0 +1,267 @@
+package libkpod
```

Comment:
Given that we're not moving config over, should I go back to the old version of libkpod.New(), in which the values for the ContainerServer are explicitly given?